bounties_api/std_bounties/client_helpers.py

```diff
@@ -47,10 +52,23 @@ def map_bounty_data(data_hash, bounty_id):
         ipfs_hash = 'invalid'
 
     data = json.loads(data_JSON)
-    metadata = data.get('meta', {})
+
+    meta = data.get('meta', {})
+
     if 'payload' in data:
         data = data.get('payload')
 
+    metadata = data.get('metadata', {})
+
+    if metadata.get('experienceLevel', '').lower().strip() in experience_levels:
```

Comment:
Looks good and a lot cleaner here. Only thing I'd recommend is taking advantage of constant files... and you can also predefine the mapping - kind of like here -> https://github.com/Bounties-Network/BountiesAPI/blob/feature-schema-updates-continued/bounties_api/notifications/constants.py#L47-L48 where we just automatically reverse map it then import that.

Also I'm confused - so is there a `metadata` and `meta` field?

Reply:
yes, there is both metadata and meta. It looks like gitcoin made some additions to their schema because it isn't updated here: https://github.com/Bounties-Network/StandardBounties/blob/master/docs/gitcoinSchemas.md

but they are adding a metadata key here: https://github.com/gitcoinco/web/blob/abad5ea606d6a9ae45343da33ee84d513af08d84/app/assets/v2/js/pages/new_bounty.js#L205-L216